serverless/aws/examples/src/main/scalajs/sttp/tapir/serverless/aws/examples/LambdaApiJsExample.scala

```diff
@@ -0,0 +1,27 @@
+package sttp.tapir.serverless.aws.examples
+
+import cats.syntax.all._
+import sttp.tapir._
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.serverless.aws.lambda._
+import sttp.tapir.serverless.aws.lambda.js._
+
+import scala.scalajs.js.annotation._
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor, Future}
+
+object LambdaApiJsExample {
+
+  implicit val ec: ExecutionContextExecutor = ExecutionContext.global
+
+  val helloEndpoint: ServerEndpoint[Unit, Unit, String, Any, Future] = endpoint.get
+    .in("api" / "hello")
+    .out(stringBody)
+    .serverLogic { _ => Future(s"Hello!".asRight[Unit]) }
+
+  val options: AwsServerOptions[Future] = AwsFutureServerOptions.default.copy(encodeResponseBody = false)
```

Comment:
What's `encodeResponseBody` and does it have to be `false` here?

Reply:
It's possible that I added it myself (or maybe not? ;) ) but even if so, I don't remember what it was doing, so some scaladocs would be useful :)

Reply:
copy-paste from JVM example ;)

Reply:
it seems like it's for determining whether or not the lambda response should be base64 encoded

Reply:
hm ... ok, let's leave it for the next iteration :)